autopush/web/webpush.py

```diff
@@ -255,6 +255,40 @@ def reject_encryption_key(self, data, original_data):
             )
 
 
+class WebPushCrypto06HeaderSchema(Schema):
+    """Validates WebPush Message Encryption
+
+    Uses draft-ietf-httpbis-encryption-encoding-06 rules for validation
+
+    """
+
+    content_encoding = fields.String(
+        required=True,
+        load_from="content-encoding",
+        validate=OneOf(["aes128gcm"])
```

Comment:
nitpick: think I've mentioned this before but we always end up w/ OneOf([onething]), should just be Equals(onething)